src/oic/oic/message.py

```diff
@@ -364,7 +370,9 @@ def verify(self, **kwargs):
                     return False
 
         if "id_token" in self:
+            _raw_id_token = self['id_token']
             self["id_token"] = verify_id_token(self, check_hash=True, **kwargs)
+            self['_raw_id_token'] = _raw_id_token
```

Comment:
Do we really need to store it? If yes, then maybe as a class attribute so it doesn't show up in the message itself?

Reply:
Yes, need to store. Since it's later going to be used as an id_token_hint.
Class attribute is a good idea.